statsmodels/stats/_lilliefors.py

```diff
@@ -87,55 +99,124 @@ def ksstat(x, cdf, alternative='two_sided', args=()):
+                               [ 25,  26,  28,  30,  35,  42]])[:,::-1] / 1000.
+
+        # also build a table for larger sample sizes
+        def f(n):
+            return np.array([.736/np.sqrt(n), .768/np.sqrt(n),
+            .805/np.sqrt(n), .886/np.sqrt(n), 1.031/np.sqrt(n)])
+
+        higher_sizes = np.array([35, 40, 45, 50, 60, 70,
+                                80, 100, 200, 500, 1000,
+                                2000, 3000, 5000, 10000, 100000], float)
+        higher_crit_lf = np.zeros([higher_sizes.shape[0], crit_lf.shape[1]-1])
+        for i in range(len(higher_sizes)):
+            higher_crit_lf[i,:] = f(higher_sizes[i])
+
+        alpha_large = alpha[:-1]
+        size_large = np.concatenate([size, higher_sizes])
+        crit_lf_large = np.vstack([crit_lf[:-4,:-1], higher_crit_lf])
+
+    elif dist == 'exp':
+        alpha = np.array([0.2,  0.15,  0.1,  0.05, 0.01])[::-1]
+        size = np.array([3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,25,30], float)
+
+        crit_lf = np.array([   [451, 479, 511, 551, 600],
+                                [396, 422, 499, 487, 548],
+                                [359, 382, 406, 442, 504],
+                                [331, 351, 375, 408, 470],
+                                [309, 327, 350, 382, 442],
+                                [291, 308, 329, 360, 419],
+                                [277, 291, 311, 341, 399],
+                                [263, 277, 295, 325, 380],
+                                [251, 264, 283, 311, 365],
+                                [241, 254, 271, 298, 351],
+                                [232, 245, 261, 287, 338],
+                                [224, 237, 252, 277, 326],
+                                [217, 229, 244, 269, 315],
+                                [211, 222, 236, 261, 306],
+                                [204, 215, 229, 253, 297],
+                                [199, 210, 223, 246, 289],
+                                [193, 204, 218, 239, 283],
+                                [188, 199, 212, 234, 278],
+                                [170, 180, 191, 210, 247],
+                                [155, 164, 174, 192, 226]])[:,::-1] / 1000.
+
+        def f(n):
+            return np.array([.86/np.sqrt(n), .91/np.sqrt(n),
+            .96/np.sqrt(n), 1.06/np.sqrt(n), 1.25/np.sqrt(n)])
+
+        higher_sizes = np.array([35, 40, 45, 50, 60, 70,
+                                80, 100, 200, 500, 1000,
+                                2000, 3000, 5000, 10000, 100000], float)
+        higher_crit_lf = np.zeros([higher_sizes.shape[0], crit_lf.shape[1]])
+        for i in range(len(higher_sizes)):
+            higher_crit_lf[i,:] = f(higher_sizes[i])
+
+        size = np.concatenate([size, higher_sizes])
+        crit_lf = np.vstack([crit_lf, higher_crit_lf])
+
 
 
     lf = TableDist(alpha, size, crit_lf)
 
     return lf
 
-lilliefors_table = get_lilliefors_table()
```

Comment:
I think we should create two instances here, lllieforse_table_norm and ..._expon